src/components/pages/events/filters/dropdown-with-two-select/dropdown-with-two-select.jsx

```diff
@@ -0,0 +1,137 @@
+import ChevronIcon from 'icons/chevron.inline.svg';
+
+const DropdownWithTwoSelect = ({
+  mainFilter,
+  secondFilter,
+  activeFilters,
+  handleFilters,
+  className,
+}) => {
+  const dropdownRef = useRef(null);
+  const buttonRef = useRef(null);
+  const [isOpen, setIsOpen] = useState(false);
+
+  const { name, items: itemsMain } = mainFilter;
+  const { items: itemsSecond } = secondFilter;
+  const valuesMain = activeFilters[mainFilter.label];
+  const valuesSecond = activeFilters[secondFilter.label];
+
+  const handleSelect = (state, value, filter) => {
+    if (state) {
+      handleFilters(filter, [...activeFilters[filter.label], value]);
+    } else {
+      handleFilters(
+        filter,
+        activeFilters[filter.label].filter((currentValue) => currentValue !== value)
+      );
+    }
+  };
+
+  const handleDropdownOutsideClick = () => {
+    setIsOpen(false);
+  };
+
+  const handleOpen = () => {
+    setIsOpen(!isOpen);
+  };
+
+  useClickOutside([dropdownRef, buttonRef], handleDropdownOutsideClick);
+
+  return (
+    <div className={clsx('relative', className)}>
+      <button
+        type="button"
+        className="group flex w-60 items-center justify-between rounded-full border border-gray-90 py-4 px-5 transition-colors duration-200 hover:border-gray-40 [@media(max-width:550px)]:w-full"
+        ref={buttonRef}
+        onClick={handleOpen}
+      >
+        <span
+          className={clsx(
+            'text-base font-medium leading-none text-gray-40 transition-colors duration-200 group-hover:text-black',
+            isOpen && '!text-black',
+            !!valuesMain.length && '!font-medium !tracking-normal !text-black'
+          )}
+        >
+          {name}
+        </span>
+        <ChevronIcon
+          className={clsx(
+            'mt-1 h-auto w-2.5 shrink-0 text-gray-40 transition-[transform,color] duration-200 group-hover:text-black',
+            isOpen && '-rotate-180',
+            (isOpen || !!valuesMain.length) && '!text-black'
+          )}
+        />
+      </button>
+      <div
+        className={clsx(
+          'absolute top-16 left-0 w-[270px] rounded border border-gray-90 bg-white drop-shadow-book transition-[opacity,visibility] duration-200 hover:drop-shadow-book [@media(max-width:550px)]:w-full',
+          isOpen ? 'visible z-10 opacity-100' : 'invisible -z-10 opacity-0'
+        )}
+        ref={dropdownRef}
+      >
+        <nav className="flex flex-col items-start justify-between px-4 py-5 md:p-3">
+          <ul className="flex flex-col">
+            {itemsMain.map(({ name }, index) => (
+              <li className="flex items-center space-x-2.5 pt-2.5 first:pt-0 md:pt-3" key={index}>
+                <Checkbox
+                  id={name}
+                  name="type"
+                  label={name}
+                  value={name}
+                  checked={valuesMain.includes(name)}
+                  onChange={(e) => handleSelect(e.target.checked, e.target.value, mainFilter)}
```

Comment:
should we pass event to the function, the get state and value on function level? 
```suggestion
                  onChange={(e) => handleSelect(e, mainFilter)}
```